modules/claims_api/app/services/claims_api/disability_compensation/docker_container_service.rb

```diff
@@ -25,6 +25,10 @@ def upload(claim_id)
                              "Successfully submitted to Docker container with response: #{evss_res}")
         # update with the evss_id returned
         auto_claim.update!(evss_id: evss_res[:claimId])
+      rescue => e
+        auto_claim.evss_response = e&.errors if e.methods.include?(:errors)
+        auto_claim.status = ClaimsApi::AutoEstablishedClaim::ERRORED
+        auto_claim.save
```

Comment:
Will we always reach this code if the PDF Generator or Form526 services fail?

Reply:
I believe the order here needs to change.  I _believe_ we would need to set it as errored first in order to make sure that happens, then the rest of the process as far as saving the evss_response, etc.  Just to be safe and, as Tyler eluded to (I _think_), make sure the errored status gets set, that is probably the #1 priority (I _believe_)